### PR TITLE
[SMALLFIX] Code style improvements

### PR DESCRIPTION
**Pull request by peisun1115**

### CODE REVIEW

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 1]
```diff
@@ -945,6 +947,8 @@
         "alluxio.worker.network.netty.file.reader.threads.max";
     public static final String WORKER_NETWORK_NETTY_FILE_WRITER_THREADS_MAX =
         "alluxio.worker.network.netty.file.writer.threads.max";
+    public static final String WORKER_NETWORK_NETTY_RPC_THREADS_MAX =
+        "alluxio.worker.network.netty.rpc.threads.max";
```
Comment (calvinjia) [inline core/common/src/main/java/alluxio/PropertyKey.java]: Could you update the documentation for this?

Comment (peisun1115) [inline core/common/src/main/java/alluxio/PropertyKey.java]: Done

**Comment by peisun1115** [comment]
@calvinjia PTAL. I've added docs.

**Comment by peisun1115** [comment]
@calvinjia PTAL? thanks

**Pull request merged**